packages/linter/src/executors/eslint/lint.impl.spec.ts

```diff
@@ -65,6 +65,7 @@ function createValidRunBuilderOptions(
     rulesdir: [],
     resolvePluginsRelativeTo: null,
     reportUnusedDisableDirectives: null,
+    printConfig: null,
```

Comment:
Let's add a test that tests this functionality.
We can mock the `calculateConfigForFile` method.

Reply:
Done :)